### PR TITLE
Dz-12

### PR DESCRIPTION
**Pull request by worldwideboi**

### CODE REVIEW

File: dz-12/src/test/java/cc/robotdreams/tests/ManAndWomanTests.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        private enum ExpectedStatus
+        {
+         PASSED,
+         FAILED
+        }
+
+        @DataProvider
+        public Object[][] DataMan() {
+            return new Object[][]{
+                    {ExpectedStatus.PASSED,"Joe","Doe", 70},
```
Comment (yurii-voronenko) [inline dz-12/src/test/java/cc/robotdreams/tests/ManAndWomanTests.java:20]: Better to use structure like `{ExpectedStatus.PASSED, new Man("Joe","Doe", 70)}`
Also as I wrote for previous homework use boundary values to test retirement

File: dz-12/src/test/java/cc/robotdreams/tests/ManAndWomanTests.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                john.setAge(50);
+                Assert.assertFalse(john.isRetired());
+            }
+
+    }
+
+    @DataProvider
+    public Object[][] DataWoman() {
+        return new Object[][]{
+                {ExpectedStatus.PASSED,"Jane", "Smith", 65},
```
Comment (yurii-voronenko) [inline dz-12/src/test/java/cc/robotdreams/tests/ManAndWomanTests.java:39]: The same as for man

**Review by yurii-voronenko — APPROVED** [review]
LGTM
